### PR TITLE
Persistent top/bottom frame display selection in left_nav

### PR DESCRIPTION
**Pull request by mdsupport**

### CODE REVIEW

File: interface/main/left_nav.php [hunk 1]
```diff
@@ -410,6 +419,10 @@ function(data) {
  $(document).ready(function (){
    getReminderCount();//
    parent.loadedFrameCount += 1;
+   for (var i = 0, len = usrval.length; i < len; i++) {
+       if (usrval[i] != "checked") toggleFrame(i+1);
+   }
+   // Following block can be removed if and when drop_bottom is eliminated
```
Comment (bradymiller) [inline interface/main/left_nav.php]: Recommend removing this setting from globals along with the pertinent code.

Comment (bradymiller) [inline interface/main/left_nav.php]: should be easy to do:
``````

Searching 11957 files for "drop_bottom"

/home/brady2/git/openemr/interface/main/left_nav.php:
  411     getReminderCount();//
  412     parent.loadedFrameCount += 1;
  413:    <?php if ($GLOBALS['drop_bottom'] ==1) { ?>   
  414     $("input[name=cb_bot]").trigger('click');
  415     toggleFrame(2);

/home/brady2/git/openemr/library/globals.inc.php:
  105                                 'gbl_pt_list_page_size',
  106                                 'gbl_pt_list_new_window',
  107:                                'drop_bottom',
  108                                 'units_of_measurement',
  109                                 'us_weight_format',
  ...
  303      ),
  304  
  305:    'drop_bottom' => array(
  306        xl('Remove Bottom Pane'),
  307        'bool',                           // data type

3 matches across 2 files
``````

**Comment by bradymiller** [comment]
Didn't test it, but the code looks very good. If remove the drop_bottom code elements, then will bring this into the codebase (assuming it tests well, which I am sure it will).

File: interface/main/left_nav.php (hunk 1 above)
Comment (mdsupport) [inline interface/main/left_nav.php]: Done.

**Comment by bradymiller** [comment]
Very nice feature. Will bring this into the codebase and thanks for the contribution!

**Pull request merged**